install/install_for_macos_linux.sh

```diff
@@ -18,12 +37,14 @@ info() {
   printf "\e[1;32m ---> \e[0m $1 \n"
 }
 
-should_install_unzip() {
-  if [[ $(command -v unzip) ]]; then
-    return 1
-  fi
+sucess() {
+  printf "\e[1;32m ---> $1 \e[0m \n"
 }
 
+if [ "$AVAIABLE_SIZE_FS" -le 10 ];then
+ warning "You may run out of space as you have only $AVAIABLE_SIZE_FS gb left on your tipi installation drive. A tipi installation typically requires between 2 and 10 gb of disk space depending on installation mode. You may change the installation location by setting the TIPI_HOME_DIR environment variable"
```

Comment:
`gb` in lower case isn't any kind of known size notations. Please stick to correct Decimal or Binary notation of sizes (depending on what `df` returns - which is decimal *I think*)